bundle/bundle-writer.go

```diff
@@ -0,0 +1,352 @@
+package bundle
```

Comment:
Please always use `gofmt` and `golint` all the time. You can configure GoLand or VSCode to run it automatically.